test/modules/template.service.js

```diff
@@ -140,6 +156,19 @@ async function getNumberOfTemplates(queryParams) {
     throw new Error(`getNumberOfTemplates - Unknown status code received: ${statusCode}`);
 }
 
+async function getNumberOfEnabledTemplates(queryParams) {
+    const { statusCode, body: templates } = await getEnabledTemplates(queryParams);
```

Comment:
Instead of adding the `getEnabledTemplates` function can't we just call `getTemplates` with the queryParams:
```
queryParams = { showEnabledOnly: true };
```